getting-started/templates/pilot-sizing.yaml

```diff
@@ -0,0 +1,891 @@
+# Default values for SystemLink pilot resources and replicas.
+# This is a YAML-formatted file.
+# Declare override values for variables.
+
+rabbitmq:
+  resources:
+    requests:
+      cpu: "100m"
+      memory: "256Mi"
+    limits:
+      memory: "512Mi"
+
+minio:
+  persistence:
+    size: 1Gi
+  provisioning:
```

Comment:
Provisioning only works if you enable minio clustering. (`mode: distributed`). That's not the default for the top-level chart or the values file in this folder. This configuration will be ignored in favor of the defaultBuckets setting.

Reply:
As per our offline discussion, I will leave this configuration.